doc-content/enterprise-only/admin-and-config/managing-business-central-configuring-githooks-proc.adoc

```diff
@@ -101,14 +103,12 @@ For example:
 
 [NOTE]
 ====
-Its not necessary to define all the exit codes in the `*.properties` file. Notifications will appear only for the exit
-codes defined in the `*.properties`.
+Its not necessary to define all the exit codes in the properties file. Notifications will appear only for the exit codes defined in the properties file.
```

Comment:
Something strange is happening in these sections on "Customizing git hook notifications" section, "Types of exit codes", and "Internationalization". It's duplicated and all pretty rough, like dev threw it in, a writer worked through it a bit, but then forgot to delete the old content. But even the new content that seems to replace it is not in great shape. Formatting is off, one step isn't even a step, etc. Have a look and see what's going on and comb through to tighten up the screws.

Reply:
Yes, there is duplication, I used the more recent doc. It did need a lot of work. I have another draft of this now. Will email you when preview is built